.github/workflows/increment-version.yml

```diff
@@ -0,0 +1,22 @@
+on:
```

Comment:
Please add Action name and also description for the job, to explain what this pipeline does

Reply:
Added a comment with a description of this pipeline. Where should I add the name?